WordPress/Docs/WP/GlobalVariablesOverrideStandard.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Global Variables Override"
+    >
+    <standard>
+    <![CDATA[
+    WordPress global variables should not be overridden.
+
+    Two WordPress global variables, `$content_width` and `$wp_cockneyreplace`, are 
+    specifically intended to be overridden by themes/plugins and are exempt from this rule.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Different variable name.">
```

Comment:
I'm not sure about this title. Maybe `Valid: Using custom variable/key names instead of WordPress globals.` or `Valid: Using custom variable/key names.` is a bit better and more descriptive? Happy to hear other suggestions you might have.

The concern that I have with `Valid: Different variable name.` is that it reads a bit vaguely and doesn't clearly communicate what makes the examples below valid.